app/models/invoice.rb

```diff
@@ -19,4 +19,14 @@ def invoice_revenue
   def self.total_invoices_revenue
     join(:invoice_items).sum("unit_price * quantity").limit(5)
   end
+
+  def self.highest_date
+    select("invoices.created_at, count(invoices.*) as date_count")
+      .order(created_at: :desc)
+      .group(:created_at)
+      .order(date_count: :desc)
```

Comment:
Took me like 3 reads of the user story to realize why you had two .orders (didn't know that was a thing). But makes total sense. Nicely done.